MyCryptoHQ/MyCrypto
# Fixed data field by auto-opening advanced tab on `data` prefilled field.

Closes #2134

### Description

This PR serves to auto-open the Advanced tab when an advanced pre-fill query is used, re-enabling sending with data fields.

### Changes

*  Added a new parameter for Query vs AdvancedQuery rendering that checks whether a `data` or `gaslimit` field is present in the url query, making it an advanced pre-filled transaction. 

### Steps to Test

1. Navigate to https://localhost:3000/account/send/?&to=0x4bbeEB066eD09B7AEd07bF39EEe0460DFa261520&value=1&data=0xadefadedafdef
2. Access your account using any of the methods we support.
3. Gas Price, Gas Limit, and Nonce should still be auto-calculated.

### Screenshots

![pre-filled-transaction-page](https://user-images.githubusercontent.com/29407814/51546792-ea52be00-1e32-11e9-8ef8-56978631f89f.PNG)
